src/main/webapp/style.css

```diff
@@ -5,3 +5,33 @@ h3 {
 .srp-container {
   display: flex;
 }
+
+#auth {
+  width: 47px;
+  height: 15px;
+  color: white;
+  background: #4d90fe;
+  border: 1px #3079ED solid;
+  padding: 5px 10px;
+  border-radius: 2px;
+  font: 400 13.3333px Arial;
+  text-align: center;
+  text-decoration: none;
+  font-size: 9pt;
+  outline: none;
+}
+
+#auth:hover {
+  border: 1px #2F5BB7 solid;
+  box-shadow: 0 1px 1px #EAEAEA, inset 0 1px 0 #5A94F1;
+  background: #3F83F1;
+}
+
+#auth:active {
+  box-shadow: inset 0 2px 5px #2370FE;   
+}
+
+.topcorner{
+  position:absolute;
```

Comment:
I don't think this requires absolute positioning. I also wonder if you might want to the logo and sign-in button on the same level. Maybe make a flexbox container for both?